sigal/themes/galleria/templates/album_items.html

```diff
@@ -0,0 +1,231 @@
+        <div class="icons">
+          {% if settings.show_map and album.show_map %}
+          <a class="gallery-map-toggle"><img src="{{ theme.url }}/img/map.png"
+            title="Show/Hide map" alt="Show/Hide Map (m)" /></a>
+          {% endif %}
+          <a id="fullscreen"><img src="{{ theme.url }}/img/fullscreen.png"
+            title="Fullscreen" alt="Fullscreen (f)" /></a>
+        </div>
+        <div id="gallery"></div>
+
+        {% if album.zip %}
+        <div id="additionnal-infos" class="row">
+          <p><a href="{{ album.zip }}"
+                title="Download a zip archive with all images">Download ZIP</a>
+          </p>
+        </div>
+        {% endif %}
+
+        {% if album.description %}
+        <div id="description">
+          {{ album.description }}
+        </div>
+        {% endif %}
+
+{% block late_js %}
```

Comment:
Just to be sure, is this loaded in the correct block (`late_js`) even if it was included in another block (`content`) ?

Reply:
Also one drawback of this is that the javascript includes and script will always be loaded in the album list pages pages, even of not needed. I  guess we could prevent that by checking if `album.medias` is not None.

Reply:
> Just to be sure, is this loaded in the correct block (`late_js`) even if it was included in another block (`content`) ?

I think so. At least it seems to work correctly in the tests I performed in Chrome and Firefox.

Reply:
> Also one drawback of this is that the javascript includes and script will always be loaded in the album list pages pages, even of not needed. I guess we could prevent that by checking if `album.medias` is not None.

I agree that this could be optimized to not load the javascript when there is no need. Checking album.medias seems like a proper way of doing that.